shakenfist/virt.py

```diff
@@ -443,22 +477,18 @@ def hard_delete(self):
     def allocate_instance_ports(self):
         with db.get_lock('attribute/instance', self.uuid, 'ports',
```

Comment:
As an aside, I think it would be better to use a lock construction method to guarantee consistent lock names.

Reply:
That's true, except this name doesn't fit with our current naming scheme. I need to fix that.